exchange/logger/log.py

```diff
@@ -0,0 +1,26 @@
+import coloredlogs
+import logging
+
+from ..meta import SingletoneMeta
```

Comment:
Лучше избегать `..` при импортах

```suggestion
from exchange.meta import SingletoneMeta
```

Reply:
исправил